examples/test.ini

```diff
@@ -8,3 +8,4 @@ name = Bob Smith       ; Spaces around '=' are stripped
 email = bob@smith.com  ; And comments (like this) ignored
 active = true          ; Test a boolean
 pi = 3.14159           ; Test a floating point number
+big = 2147483648       ; Test a big integer
```

Comment:
Let's use a larger value for the test integer here. 2147483648 is more than a 32-bit signed long can hold, but fits fine in a 32-bit unsigned long. Maybe 1000000000000?

Reply:
That was an oversight on my part. Fixed in 0bc41b98f3c28f00c1c264c3520418656e764641